src/FindApprenticeshipJobs/SFA.DAS.FindApprenticeshipJobs.Api.UnitTests/Controllers/WhenPostingCandidateStatus.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.FindApprenticeshipJobs.Api.Models;
+using SFA.DAS.FindApprenticeshipJobs.Application.Commands.Candidates;
+using SFA.DAS.Testing.AutoFixture;
+using System.Net;
+
+namespace SFA.DAS.FindApprenticeshipJobs.Api.UnitTests.Controllers
+{
+    [TestFixture]
+    public class WhenPostingCandidateStatus
+    {
+        [Test, MoqAutoData]
+        public async Task Then_Returns_Post_Response(
+            string govIdentifier,
+            CandidateUpdateStatusRequest model,
+            [Frozen] Mock<IMediator> mediator,
+            [Greedy] Api.Controllers.CandidatesController controller)
+        {
+            var actual = await controller.UpdateStatus(govIdentifier, model);
```

Comment:
might be good to also assert/verify that the params being passed go to the mediator handler correctly

Reply:
Added extra assert/verify to check the mediator is called with same params.